src/vuestic-theme/vuestic-components/va-input/VaInput.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <va-input-wrapper
+    class="va-input"
+    :class="{ 'va-input-wrapper--focused': isFocused }"
+    :disabled="disabled"
+    :error="error"
+    :success="success"
+    :messages="messages"
+    :error-messages="errorMessages"
+  >
+    <slot name="prepend" slot="prepend"/>
+    <div
+      class="va-input__slot d-flex">
+      <label
+        :style="labelStyles"
+        aria-hidden="true"
+        class="va-input__slot__label title"
+      >
+        {{ label }}
+      </label>
+      <input
+        class="py-1 px-2"
+        :type="type"
+        :placeholder="placeholder"
+        :disabled="disabled"
+        :readonly="readonly"
+        v-model="currentValue"
+        @input="$emit('input', currentValue)"
+        @focus="updateFocusState(true)"
+        @blur="updateFocusState(false)"
+      >
+    </div>
+    <va-icon
+      @click.native="clearContent()"
+      v-if="removable"
+      slot="append"
```

Comment:
What happens if I have a cleareable input and also add something to "append" slot?

Reply:
I would say close icon should stay. As `closable` is not default state.